api/src/main/java/com/velocitypowered/api/proxy/Player.java

```diff
@@ -45,6 +46,20 @@
    */
   String getUsername();
 
+  /**
+   * The locale the velocity proxy is translating its messages to. By default {@link PlayerSettings#getLocale()}
+   *
+   * <p>Can be null when the settings have not been initialized yet.</p>
+   *
+   * @return the locale.
+   *
```

Comment:
Code style: don't have an empty line here.

Reply:
The line was removed.